src/Administration/Controller/NotificationController.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Shopware\Administration\Controller;
```

Comment:
I think, this should also be moved :thinking:

Reply:
can we do that without a bc layer? It's not internal

Reply:
move everything into a core controller and let this one extend from the core one?

Reply:
done